commands/info/command.go

```diff
@@ -261,6 +261,7 @@ func getEnvironmentVariables() map[string]string {
 	vars := []string{
 		"GSCTL_CAFILE",
 		"GSCTL_CAPATH",
+        "GSCTL_CONFIG_DIR",
```

Comment:
Use a tab character here

```suggestion
	"GSCTL_CONFIG_DIR",
```

Reply:
Whoops, sorry about that.